Formula/frps.rb

```diff
@@ -0,0 +1,72 @@
+class Frps < Formula
+  desc "This is the server of frp"
```

Comment:
Maybe this is a clearer description?
```suggestion
  desc "A fast reverse proxy server to expose a local server to the internet"
```

Reply:
It contains some more of the project description.

Reply:
Thanks for suggestion! 
But   desc "A fast reverse proxy server to expose a local server to the internet"
is not pass the test

    brew audit --strict --online frps
    frps:
      * C: 2: col 9: Description shouldn't start with an indefinite article i.e. "A"
    Error: 1 problem in 1 formula detected

And because this the the server part of frp, so I change desc to 

    desc "Server app of fast reverse proxy to expose a local server to the internet"